### PR TITLE
Cleanup rh-ruby25 RPMs on upgrade for nightly and Foreman 2.5+

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: roles/cleanup_rh_ruby25/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+- name: Remove rh-ruby25 packages
+  package:
+    state: absent
+    name: rh-ruby25*
```
Comment (ekohl) [inline roles/cleanup_rh_ruby25/tasks/main.yml:5]: You can optimize a bit by adding `when EL7` to it.

**Review by ekohl — COMMENTED** [review]
I'm wondering if it makes sense to have some `post_upgrade` role with multiple tasks. Does that make sense?

**Comment by ehelms** [comment]
> I'm wondering if it makes sense to have some `post_upgrade` role with multiple tasks. Does that make sense?

I started down this approach but then realized that the tasks are often version specific, so you have to start burying that version info into the module, or through multiple parameters per item and it could spiral out of hand.

File: roles/cleanup_rh_ruby25/tasks/main.yml (hunk 1 above)
Comment (ehelms) [reply roles/cleanup_rh_ruby25/tasks/main.yml:5]: Done.

**Comment by evgeni** [comment]
I think nightly will never be intermediate ;)

**Comment by ehelms** [comment]
> I think nightly will never be intermediate ;)

Touche sir, touche.

**Review by ekohl — APPROVED** [review]

**Pull request merged**